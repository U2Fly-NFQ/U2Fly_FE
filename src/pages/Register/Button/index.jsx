export function Button({
  children,
  type,
  loading,
  disabled,
  className,
  style,
}) {
  return (
    <button type={type} disabled={disabled} style={style} className={className}>
      {children}
    </button>
  )
}
