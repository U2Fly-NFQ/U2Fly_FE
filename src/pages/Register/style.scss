@use '../../assets/scss/abstracts' as a;

.register-page {
  background-color: #fff;

  .content {
    padding: 400px 0;
    position: relative;

    .box {
      background: #f3f6fd;
      padding: 45px 20px;
      position: absolute;
      top: -200px;
      left: 0;
      right: 0;
      border-radius: 12px;
      box-shadow: a.$box-shadow;

      h2,
      h3 {
        margin: 0;
        text-align: center;
      }

      h3 {
        font-weight: 400;
        color: a.$paragraph-color;
      }

      h2 {
        padding-top: 15px;
        font-size: 36px;
        font-weight: 500;
        line-height: 40px;
      }

      .form {
        padding: 0 100px;
        @media screen and (max-width: 992px) {
          padding-left: 0;
          padding-right: 0;
        }
        padding-top: 50px;

        .form-group {
          margin-bottom: 20px;
          text-align: right;

          .form-control {
            input {
              border: none;
              outline: none;
              width: 100%;
            }

            input:focus {
              box-shadow: none;
            }

            span {
              border: none;
              box-shadow: none;
            }

            .ant-form-item-explain-error {
              margin-top: 10px;
              text-align: start;
            }

            height: 55px;
            border: 2px dashed #dddddd75;
            padding: 10px 22px;
            font-size: 16px;
            box-shadow: 0px 1px 13px 0px #0000000d;
            display: block;
            width: 100%;
            font-weight: 400;
            line-height: 1.5;
            color: #212529;
            background-color: #fff;
            background-clip: padding-box;
            appearance: none;
            border-radius: 0.25rem;
            transition: border-color 0.15s ease-in-out,
              box-shadow 0.15s ease-in-out;

            &:focus-within {
              background-color: #fff;
              border: 2px dashed a.$main-color;
              outline: none;
              box-shadow: none;
            }
          }

          a {
            display: block;
            color: a.$paragraph-color;
            padding-top: 10px;

            &:hover {
              color: a.$main-color;
            }
          }
        }

        .form-submit {
          display: flex;
          justify-content: center;
          padding-top: 20px;

          .btn {
            display: inline-block;
            font-weight: 400;
            line-height: 1.5;
            text-align: center;
            text-decoration: none;
            vertical-align: middle;
            cursor: pointer;
            user-select: none;
            font-size: 16px;
            border-radius: 5px;
            box-shadow: none;
            overflow: hidden;
            white-space: nowrap;
            position: relative;
            z-index: 0;

            &.btn-primary {
              color: a.$white-color;
              background-color: a.$main-color;
              transition: a.$transition;
              overflow: hidden;
              white-space: nowrap;
              position: relative;
              z-index: 0;
              border: none;
            }

            &.btn-md {
              padding: 12px 35px;
              font-size: 18px;
            }

            &::before {
              content: '';
              position: absolute;
              top: 0;
              bottom: 0;
              left: 50%;
              width: 580px;
              height: 550px;
              margin: auto;
              background: var(--ant-primary-5);
              border-radius: 50%;
              z-index: -1;
              transform-origin: top center;
              transform: translateX(-50%) translateY(-5%) scale(0.4);
              transition: a.$transition;
            }

            &:hover::before {
              transition: transform 1s;
              transform: translateX(-50%) translateY(0) scale(1);
              transform-origin: bottom center;
              border: none;
            }
          }
        }

        .switch {
          text-align: center;
          padding-top: 20px;

          p {
            font-size: 16px;
            line-height: 28px;
            color: a.$paragraph-color;
            font-weight: 400;
            margin-bottom: 0;

            a {
              text-decoration: none;
              transition: all 0.3s ease-in-out 0.1s;
              outline: 0 !important;
              color: a.$main-color;
            }
          }
        }
      }
    }
  }
}
