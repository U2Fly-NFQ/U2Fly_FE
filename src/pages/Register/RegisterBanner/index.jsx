import { useTranslation } from 'react-i18next'
import {
  motion,
  useMotionValue,
  useTransform,
  useViewportScroll,
} from 'framer-motion'
import { Typography } from 'antd'
import './style.scss'

const { Title, Text } = Typography

function RegisterBanner() {
  const { t } = useTranslation()

  const x = useMotionValue(0)
  const opacity = useTransform(x, [-100, 0, 100], [0, 1, 0])
  const { scrollYProgress } = useViewportScroll()

  const variants = {
    hidden: { opacity: 0 },
    visible: { opacity: 1 },
  }

  return (
    <section className="flight-list-banner">
      <div className="grid wide">
        <div className="flight-list-banner__text">
          <motion.div
            initial={{
              opacity: 0,
              x: -200,
            }}
            animate={{ opacity: 1, x: 0 }}
            transition={{ duration: 1 }}
          >
            <Title level={1}>{t('Register your account')}</Title>
          </motion.div>
          <motion.div
            initial={{
              opacity: 0,
              x: 200,
            }}
            animate={{ opacity: 1, x: 0 }}
            transition={{ duration: 1 }}
          >
            <Title level={3}>To join a whole new world</Title>
          </motion.div>
        </div>
      </div>
    </section>
  )
}

export default RegisterBanner
