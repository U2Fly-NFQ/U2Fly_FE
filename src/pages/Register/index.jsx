import { Row, Col, Typography } from 'antd'
import { LoginBanner } from '../../components'
import './style.scss'

const { Title } = Typography

const Register = () => {
  const onFinish = (values) => {
    console.log('Success:', values)
  }

  const onFinishFailed = (errorInfo) => {
    console.log('Failed:', errorInfo)
  }

  return (
    <>
      <div className="register-page">
        <LoginBanner />
        <div className="grid wide">
          <Row justify="center">
            <Col lg={16}>
              <div className="content">
                <div className="box">
                  <Title level={3}>To join a whole new world</Title>
                  <Title level={2}>Register your account</Title>
                  <form action="#" className="form">
                    <div className="form-group">
                      <input
                        type="text"
                        className="form-control"
                        placeholder="Your name"
                      />
                    </div>
                    <div className="form-group">
                      <input
                        type="text"
                        className="form-control"
                        placeholder="Your mail"
                      />
                    </div>
                    <div className="form-group">
                      <input
                        type="text"
                        className="form-control"
                        placeholder="Your phone"
                      />
                    </div>
                    <div className="form-group">
                      <input
                        type="password"
                        className="form-control"
                        placeholder="Enter password"
                      />
                      <a href="forgot-password.html">Forgot password?</a>
                    </div>
                    <div className="form-submit">
                      <button className="btn btn-primary btn-md">Register</button>
                    </div>
                    <div className="switch">
                      <p>
                        Dont have an account?{' '}
                        <a href="register.html">Login now</a>
                      </p>
                    </div>
                  </form>
                </div>
              </div>
            </Col>
          </Row>
        </div>
      </div>
    </>
  )
}

export default Register
