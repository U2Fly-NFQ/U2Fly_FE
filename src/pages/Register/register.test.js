import { I18nextProvider } from 'react-i18next'
import i18n from '../../translations'
import React, { Suspense } from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import { render, fireEvent, screen } from '@testing-library/react'

import Register from './'
import renderer from 'react-test-renderer'

describe('Register page', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <I18nextProvider i18n={i18n}>
          <Router>
            <Suspense>
              <Register />
            </Suspense>
          </Router>
        </I18nextProvider>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  it('form validate correctly', async () => {
    render(
      <I18nextProvider i18n={i18n}>
        <Router>
          <Suspense>
            <Register />
          </Suspense>
        </Router>
      </I18nextProvider>
    )

    const testData = {
      idNum: '123123123',
      name: 'Johnny Depp',
      genderRadio: 'male',
      birthDay: '2022-09-17',
      address: 'Can Tho',
      email: 'nfq@asisa',
      password: '123456',
      confirmPassword: '123456',
    }

    const idNum = await screen.findByPlaceholderText('ID number')
    const name = await screen.findByPlaceholderText('Display name')
    const genderRadio = await screen.findByLabelText('Male')
    const birthDay = await screen.findByTestId('birthday-picker')
    const address = await screen.findByPlaceholderText('Address')
    const email = await screen.findByPlaceholderText('Email')
    const password = await screen.findByPlaceholderText('Password')
    const confirmPassword = await screen.findByPlaceholderText(
      'Confirm password'
    )

    fireEvent.change(idNum, {
      target: {
        value: testData.idNum,
      },
    })
    expect(idNum.value).toBe(testData.idNum)

    fireEvent.change(name, {
      target: {
        value: testData.name,
      },
    })
    expect(name.value).toBe(testData.name)

    fireEvent.change(genderRadio, {
      target: {
        value: testData.genderRadio,
      },
    })
    expect(genderRadio.value).toBe(testData.genderRadio)

    fireEvent.change(password, {
      target: {
        value: testData.password,
      },
    })
    expect(password.value).toBe(testData.password)

    fireEvent.change(confirmPassword, {
      target: {
        value: testData.confirmPassword,
      },
    })
    expect(confirmPassword.value).toBe(testData.confirmPassword)
  })
})
