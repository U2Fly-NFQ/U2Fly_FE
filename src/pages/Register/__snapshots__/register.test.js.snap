// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register page  renders correctly 1`] = `
<div
  className="register-page"
>
  <section
    className="flight-list-banner"
  >
    <div
      className="grid wide"
    >
      <div
        className="flight-list-banner__text"
      >
        <div
          style={
            Object {
              "opacity": 0,
              "transform": "translateX(-200px) translateZ(0)",
            }
          }
        >
          <h1
            className="ant-typography"
            onClick={null}
            style={
              Object {
                "WebkitLineClamp": undefined,
              }
            }
          >
            Register your account
          </h1>
        </div>
        <div
          style={
            Object {
              "opacity": 0,
              "transform": "translateX(200px) translateZ(0)",
            }
          }
        >
          <h3
            className="ant-typography"
            onClick={null}
            style={
              Object {
                "WebkitLineClamp": undefined,
              }
            }
          >
            To join a whole new world
          </h3>
        </div>
      </div>
    </div>
  </section>
  <div
    className="grid wide"
  >
    <div
      className="ant-row ant-row-center"
      style={Object {}}
    >
      <div
        className="ant-col ant-col-xs-22 ant-col-md-16"
        style={Object {}}
      >
        <div
          className="content"
        >
          <div
            className="box"
          >
            <h2
              className="ant-typography"
              onClick={null}
              style={
                Object {
                  "WebkitLineClamp": undefined,
                }
              }
            >
              Register
            </h2>
            <form
              autoComplete="off"
              className="ant-form ant-form-horizontal form"
              onReset={[Function]}
              onSubmit={[Function]}
            >
              <div
                className="form-group"
              >
                <div
                  className="ant-row ant-form-item form-control"
                  style={Object {}}
                >
                  <div
                    className="ant-col ant-form-item-control"
                    style={Object {}}
                  >
                    <div
                      className="ant-form-item-control-input"
                    >
                      <div
                        className="ant-form-item-control-input-content"
                      >
                        <input
                          className="ant-input"
                          id="idno"
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          placeholder="ID number"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="form-group"
              >
                <div
                  className="ant-row ant-form-item form-control"
                  style={Object {}}
                >
                  <div
                    className="ant-col ant-form-item-control"
                    style={Object {}}
                  >
                    <div
                      className="ant-form-item-control-input"
                    >
                      <div
                        className="ant-form-item-control-input-content"
                      >
                        <input
                          className="ant-input"
                          id="name"
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          placeholder="Display name"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="form-group"
              >
                <div
                  className="ant-row ant-form-item"
                  style={
                    Object {
                      "textAlign": "left",
                    }
                  }
                >
                  <div
                    className="ant-col ant-form-item-label"
                    style={Object {}}
                  >
                    <label
                      className="ant-form-item-required"
                      htmlFor="gender"
                      title="Gender"
                    >
                      Gender
                    </label>
                  </div>
                  <div
                    className="ant-col ant-form-item-control"
                    style={Object {}}
                  >
                    <div
                      className="ant-form-item-control-input"
                    >
                      <div
                        className="ant-form-item-control-input-content"
                      >
                        <div
                          className="ant-radio-group ant-radio-group-outline"
                          id="gender"
                        >
                          <label
                            className="ant-radio-wrapper ant-radio-wrapper-in-form-item"
                          >
                            <span
                              className="ant-radio"
                              style={Object {}}
                            >
                              <input
                                checked={false}
                                className="ant-radio-input"
                                onBlur={[Function]}
                                onChange={[Function]}
                                onFocus={[Function]}
                                onKeyDown={[Function]}
                                onKeyPress={[Function]}
                                onKeyUp={[Function]}
                                type="radio"
                                value="male"
                              />
                              <span
                                className="ant-radio-inner"
                              />
                            </span>
                            <span>
                               Male 
                            </span>
                          </label>
                          <label
                            className="ant-radio-wrapper ant-radio-wrapper-in-form-item"
                          >
                            <span
                              className="ant-radio"
                              style={Object {}}
                            >
                              <input
                                checked={false}
                                className="ant-radio-input"
                                onBlur={[Function]}
                                onChange={[Function]}
                                onFocus={[Function]}
                                onKeyDown={[Function]}
                                onKeyPress={[Function]}
                                onKeyUp={[Function]}
                                type="radio"
                                value="female"
                              />
                              <span
                                className="ant-radio-inner"
                              />
                            </span>
                            <span>
                               Female 
                            </span>
                          </label>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="form-group"
              >
                <div
                  className="ant-row ant-form-item"
                  style={
                    Object {
                      "textAlign": "left",
                    }
                  }
                >
                  <div
                    className="ant-col ant-form-item-label"
                    style={Object {}}
                  >
                    <label
                      className="ant-form-item-required"
                      htmlFor="birthday"
                      title="Birth Date"
                    >
                      Birth Date
                    </label>
                  </div>
                  <div
                    className="ant-col ant-form-item-control"
                    style={Object {}}
                  >
                    <div
                      className="ant-form-item-control-input"
                    >
                      <div
                        className="ant-form-item-control-input-content"
                      >
                        <div
                          className="ant-picker"
                          onMouseUp={[Function]}
                        >
                          <div
                            className="ant-picker-input"
                          >
                            <input
                              autoComplete="off"
                              disabled={false}
                              id="birthday"
                              onBlur={[Function]}
                              onChange={[Function]}
                              onFocus={[Function]}
                              onKeyDown={[Function]}
                              onMouseDown={[Function]}
                              placeholder="Select date"
                              readOnly={true}
                              size={12}
                              title=""
                              value=""
                            />
                            <span
                              className="ant-picker-suffix"
                            >
                              <span
                                aria-label="calendar"
                                className="anticon anticon-calendar"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="calendar"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="64 64 896 896"
                                  width="1em"
                                >
                                  <path
                                    d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
                                  />
                                </svg>
                              </span>
                            </span>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="form-group"
              >
                <div
                  className="ant-row ant-form-item form-control"
                  style={Object {}}
                >
                  <div
                    className="ant-col ant-form-item-control"
                    style={Object {}}
                  >
                    <div
                      className="ant-form-item-control-input"
                    >
                      <div
                        className="ant-form-item-control-input-content"
                      >
                        <input
                          className="ant-input"
                          id="address"
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          placeholder="Address"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="form-group"
              >
                <div
                  className="ant-row ant-form-item form-control"
                  style={Object {}}
                >
                  <div
                    className="ant-col ant-form-item-control"
                    style={Object {}}
                  >
                    <div
                      className="ant-form-item-control-input"
                    >
                      <div
                        className="ant-form-item-control-input-content"
                      >
                        <input
                          className="ant-input"
                          id="email"
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          placeholder="Email"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="form-group"
              >
                <div
                  className="ant-row ant-form-item form-control"
                  style={Object {}}
                >
                  <div
                    className="ant-col ant-form-item-control"
                    style={Object {}}
                  >
                    <div
                      className="ant-form-item-control-input"
                    >
                      <div
                        className="ant-form-item-control-input-content"
                      >
                        <span
                          className="ant-input-affix-wrapper ant-input-password"
                          onMouseDown={[Function]}
                        >
                          <input
                            action="click"
                            className="ant-input"
                            hidden={null}
                            id="password"
                            onBlur={[Function]}
                            onChange={[Function]}
                            onFocus={[Function]}
                            onKeyDown={[Function]}
                            placeholder="Password"
                            style={null}
                            type="password"
                            value=""
                          />
                          <span
                            className="ant-input-suffix"
                          >
                            <span
                              aria-label="eye-invisible"
                              className="anticon anticon-eye-invisible ant-input-password-icon"
                              onClick={[Function]}
                              onMouseDown={[Function]}
                              onMouseUp={[Function]}
                              role="img"
                              tabIndex={-1}
                            >
                              <svg
                                aria-hidden="true"
                                data-icon="eye-invisible"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M942.2 486.2Q889.47 375.11 816.7 305l-50.88 50.88C807.31 395.53 843.45 447.4 874.7 512 791.5 684.2 673.4 766 512 766q-72.67 0-133.87-22.38L323 798.75Q408 838 512 838q288.3 0 430.2-300.3a60.29 60.29 0 000-51.5zm-63.57-320.64L836 122.88a8 8 0 00-11.32 0L715.31 232.2Q624.86 186 512 186q-288.3 0-430.2 300.3a60.3 60.3 0 000 51.5q56.69 119.4 136.5 191.41L112.48 835a8 8 0 000 11.31L155.17 889a8 8 0 0011.31 0l712.15-712.12a8 8 0 000-11.32zM149.3 512C232.6 339.8 350.7 258 512 258c54.54 0 104.13 9.36 149.12 28.39l-70.3 70.3a176 176 0 00-238.13 238.13l-83.42 83.42C223.1 637.49 183.3 582.28 149.3 512zm246.7 0a112.11 112.11 0 01146.2-106.69L401.31 546.2A112 112 0 01396 512z"
                                />
                                <path
                                  d="M508 624c-3.46 0-6.87-.16-10.25-.47l-52.82 52.82a176.09 176.09 0 00227.42-227.42l-52.82 52.82c.31 3.38.47 6.79.47 10.25a111.94 111.94 0 01-112 112z"
                                />
                              </svg>
                            </span>
                          </span>
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="form-group"
              >
                <div
                  className="ant-row ant-form-item form-control"
                  style={Object {}}
                >
                  <div
                    className="ant-col ant-form-item-control"
                    style={Object {}}
                  >
                    <div
                      className="ant-form-item-control-input"
                    >
                      <div
                        className="ant-form-item-control-input-content"
                      >
                        <span
                          className="ant-input-affix-wrapper ant-input-password"
                          onMouseDown={[Function]}
                        >
                          <input
                            action="click"
                            className="ant-input"
                            hidden={null}
                            id="confirm"
                            onBlur={[Function]}
                            onChange={[Function]}
                            onFocus={[Function]}
                            onKeyDown={[Function]}
                            placeholder="Confirm password"
                            style={null}
                            type="password"
                            value=""
                          />
                          <span
                            className="ant-input-suffix"
                          >
                            <span
                              aria-label="eye-invisible"
                              className="anticon anticon-eye-invisible ant-input-password-icon"
                              onClick={[Function]}
                              onMouseDown={[Function]}
                              onMouseUp={[Function]}
                              role="img"
                              tabIndex={-1}
                            >
                              <svg
                                aria-hidden="true"
                                data-icon="eye-invisible"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M942.2 486.2Q889.47 375.11 816.7 305l-50.88 50.88C807.31 395.53 843.45 447.4 874.7 512 791.5 684.2 673.4 766 512 766q-72.67 0-133.87-22.38L323 798.75Q408 838 512 838q288.3 0 430.2-300.3a60.29 60.29 0 000-51.5zm-63.57-320.64L836 122.88a8 8 0 00-11.32 0L715.31 232.2Q624.86 186 512 186q-288.3 0-430.2 300.3a60.3 60.3 0 000 51.5q56.69 119.4 136.5 191.41L112.48 835a8 8 0 000 11.31L155.17 889a8 8 0 0011.31 0l712.15-712.12a8 8 0 000-11.32zM149.3 512C232.6 339.8 350.7 258 512 258c54.54 0 104.13 9.36 149.12 28.39l-70.3 70.3a176 176 0 00-238.13 238.13l-83.42 83.42C223.1 637.49 183.3 582.28 149.3 512zm246.7 0a112.11 112.11 0 01146.2-106.69L401.31 546.2A112 112 0 01396 512z"
                                />
                                <path
                                  d="M508 624c-3.46 0-6.87-.16-10.25-.47l-52.82 52.82a176.09 176.09 0 00227.42-227.42l-52.82 52.82c.31 3.38.47 6.79.47 10.25a111.94 111.94 0 01-112 112z"
                                />
                              </svg>
                            </span>
                          </span>
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="form-submit"
              >
                <button
                  className="btn btn-primary btn-md"
                  type="submit"
                >
                  Register
                </button>
              </div>
              <div
                className="switch"
              >
                <p>
                  Already have an account?
                   
                  <a
                    href="/login"
                    onClick={[Function]}
                  >
                    Login now
                  </a>
                </p>
              </div>
            </form>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
